src/deploy/functions/runtimes/discovery/v1alpha1.ts

```diff
@@ -119,7 +121,7 @@ function parseEndpoints(
   for (const region of ep.region || [defaultRegion]) {
     let triggered: backend.Triggered;
     if (backend.isEventTriggered(ep)) {
-      requireKeys(prefix + ".eventTrigger", ep.eventTrigger, "eventType", "eventFilters");
+      requireKeys(prefix + ".eventTrigger", ep.eventTrigger, "eventType");
```

Comment:
I don't think we don't have to relax `eventFilters` being an optional field anymore?

Reply:
Right, reverted.